contracts/helpers/TransferHelper.sol

```diff
@@ -3,8 +3,18 @@ pragma solidity >=0.8.7;
 
 import "./TransferHelperStructs.sol";
 
+// prettier-ignore
+import {
+    ERC721TokenReceiver
```

Comment:
We should just declare an interface locally rather than importing it from a library, it’s just a single function